### ISSUE TITLE
[PRO] Queries with an invalid cursor can raise a JSON::ParserError exception

### ISSUE THREAD

**Issue by such** [rmosolgo/graphql-ruby#3735]
**Describe the bug**

When using a stable cursor implementation,  queries with an invalid cursor can raise a `JSON::ParserError` exception.

**Versions**

`graphql` version: 1.12.21
`graphql-pro` version: 1.20.2
`rails` (or other framework):  6.1.4.1

**Steps to reproduce**

```ruby
require "bundler/inline"

gemfile do
  gem "graphql", "1.12.21"
  gem "graphql-pro", "1.20.2"
  gem "rails", '~> 6.1.4.1'
  gem "sqlite3"
end

require "active_record"

ActiveRecord::Base.configurations = {
  primary: { adapter: "sqlite3", database: "ar_test.db" },
}
ActiveRecord::Base.establish_connection(:primary)

ActiveRecord::Schema.define do
  self.verbose = false
  create_table :things, force: true do |t|
    t.column :name, :string
  end
end

class Thing < ActiveRecord::Base
end

Thing.create!(name: "Hat")
Thing.create!(name: "Lamp")
Thing.create!(name: "Tree")

class Schema < GraphQL::Schema
  class Thing < GraphQL::Schema::Object
    field :name, String, null: false
  end

  class Query < GraphQL::Schema::Object
    field :things, Thing.connection_type, null: false

    def things
      ::Thing.all
    end
  end

  query(Query)

  connections.add(ActiveRecord::Relation, GraphQL::Pro::SqliteStableRelationConnection)
end

ActiveRecord::Base.logger = Logger.new(STDOUT)

pp Schema.execute("query q($after: String!) { things(after: $after) { nodes { name } } }", variables: { after: 'WyIyNmEzODU1YS0' }).to_h
```

**Expected behavior**

A clean error should be sent to the client

**Actual behavior**

An exception is raised and a 500 error is sent to the client

Place full backtrace here (if a Ruby exception is involved):

<details>
<summary>Click to view exception backtrace</summary>

```
/Users/adrien/.rbenv/versions/3.0.2/lib/ruby/3.0.0/json/common.rb:216:in `parse': 439: unexpected token at '"26a3855a-' (JSON::ParserError)
	from /Users/adrien/.rbenv/versions/3.0.2/lib/ruby/3.0.0/json/common.rb:216:in `parse'
	from /Users/adrien/.rbenv/versions/3.0.2/lib/ruby/gems/3.0.0/gems/graphql-pro-1.20.2/lib/graphql/pro/relation_connection/condition.rb:35:in `build_condition'
	from /Users/adrien/.rbenv/versions/3.0.2/lib/ruby/gems/3.0.0/gems/graphql-pro-1.20.2/lib/graphql/pro/relation_connection/filter.rb:22:in `filter_relation'
	from /Users/adrien/.rbenv/versions/3.0.2/lib/ruby/gems/3.0.0/gems/graphql-pro-1.20.2/lib/graphql/pro/stable_relation_connection.rb:100:in `load_nodes_by_value'
	from /Users/adrien/.rbenv/versions/3.0.2/lib/ruby/gems/3.0.0/gems/graphql-pro-1.20.2/lib/graphql/pro/stable_relation_connection.rb:62:in `load_nodes'
	from /Users/adrien/.rbenv/versions/3.0.2/lib/ruby/gems/3.0.0/gems/graphql-1.12.21/lib/graphql/pagination/relation_connection.rb:9:in `nodes'
	from /Users/adrien/.rbenv/versions/3.0.2/lib/ruby/gems/3.0.0/gems/graphql-1.12.21/lib/graphql/pagination/connection.rb:147:in `edge_nodes'
	from /Users/adrien/.rbenv/versions/3.0.2/lib/ruby/gems/3.0.0/gems/graphql-1.12.21/lib/graphql/types/relay/connection_behaviors.rb:138:in `nodes'
	from /Users/adrien/.rbenv/versions/3.0.2/lib/ruby/gems/3.0.0/gems/graphql-1.12.21/lib/graphql/schema/field.rb:726:in `public_send'
	from /Users/adrien/.rbenv/versions/3.0.2/lib/ruby/gems/3.0.0/gems/graphql-1.12.21/lib/graphql/schema/field.rb:726:in `block in public_send_field'
	from /Users/adrien/.rbenv/versions/3.0.2/lib/ruby/gems/3.0.0/gems/graphql-1.12.21/lib/graphql/schema/field.rb:767:in `block in with_extensions'
	from /Users/adrien/.rbenv/versions/3.0.2/lib/ruby/gems/3.0.0/gems/graphql-1.12.21/lib/graphql/schema/field.rb:797:in `block (2 levels) in run_extensions_before_resolve'
	from /Users/adrien/.rbenv/versions/3.0.2/lib/ruby/gems/3.0.0/gems/graphql-1.12.21/lib/graphql/schema/field.rb:800:in `run_extensions_before_resolve'
	from /Users/adrien/.rbenv/versions/3.0.2/lib/ruby/gems/3.0.0/gems/graphql-1.12.21/lib/graphql/schema/field.rb:797:in `block in run_extensions_before_resolve'
	from /Users/adrien/.rbenv/versions/3.0.2/lib/ruby/gems/3.0.0/gems/graphql-1.12.21/lib/graphql/schema/field_extension.rb:50:in `resolve'
	from /Users/adrien/.rbenv/versions/3.0.2/lib/ruby/gems/3.0.0/gems/graphql-1.12.21/lib/graphql/schema/field.rb:790:in `run_extensions_before_resolve'
	from /Users/adrien/.rbenv/versions/3.0.2/lib/ruby/gems/3.0.0/gems/graphql-1.12.21/lib/graphql/schema/field.rb:766:in `with_extensions'
	from /Users/adrien/.rbenv/versions/3.0.2/lib/ruby/gems/3.0.0/gems/graphql-1.12.21/lib/graphql/schema/field.rb:706:in `public_send_field'
	from /Users/adrien/.rbenv/versions/3.0.2/lib/ruby/gems/3.0.0/gems/graphql-1.12.21/lib/graphql/schema/field.rb:620:in `block in resolve'
	from /Users/adrien/.rbenv/versions/3.0.2/lib/ruby/gems/3.0.0/gems/graphql-1.12.21/lib/graphql/schema.rb:116:in `after_lazy'
	from /Users/adrien/.rbenv/versions/3.0.2/lib/ruby/gems/3.0.0/gems/graphql-1.12.21/lib/graphql/schema/field.rb:618:in `resolve'
	from /Users/adrien/.rbenv/versions/3.0.2/lib/ruby/gems/3.0.0/gems/graphql-1.12.21/lib/graphql/execution/interpreter/runtime.rb:508:in `block (4 levels) in evaluate_selection_with_args'
	from /Users/adrien/.rbenv/versions/3.0.2/lib/ruby/gems/3.0.0/gems/graphql-1.12.21/lib/graphql/tracing.rb:66:in `trace'
	from /Users/adrien/.rbenv/versions/3.0.2/lib/ruby/gems/3.0.0/gems/graphql-1.12.21/lib/graphql/execution/interpreter/runtime.rb:507:in `block (3 levels) in evaluate_selection_with_args'
	from /Users/adrien/.rbenv/versions/3.0.2/lib/ruby/gems/3.0.0/gems/graphql-1.12.21/lib/graphql/query.rb:366:in `block in with_error_handling'
	from /Users/adrien/.rbenv/versions/3.0.2/lib/ruby/gems/3.0.0/gems/graphql-1.12.21/lib/graphql/execution/errors.rb:107:in `with_error_handling'
	from /Users/adrien/.rbenv/versions/3.0.2/lib/ruby/gems/3.0.0/gems/graphql-1.12.21/lib/graphql/query.rb:365:in `with_error_handling'
	from /Users/adrien/.rbenv/versions/3.0.2/lib/ruby/gems/3.0.0/gems/graphql-1.12.21/lib/graphql/execution/interpreter/runtime.rb:506:in `block (2 levels) in evaluate_selection_with_args'
	from /Users/adrien/.rbenv/versions/3.0.2/lib/ruby/gems/3.0.0/gems/graphql-1.12.21/lib/graphql/execution/interpreter/runtime.rb:788:in `resolve_with_directives'
	from /Users/adrien/.rbenv/versions/3.0.2/lib/ruby/gems/3.0.0/gems/graphql-1.12.21/lib/graphql/execution/interpreter/runtime.rb:503:in `block in evaluate_selection_with_args'
	from /Users/adrien/.rbenv/versions/3.0.2/lib/ruby/gems/3.0.0/gems/graphql-1.12.21/lib/graphql/execution/interpreter/runtime.rb:887:in `after_lazy'
	from /Users/adrien/.rbenv/versions/3.0.2/lib/ruby/gems/3.0.0/gems/graphql-1.12.21/lib/graphql/execution/interpreter/runtime.rb:437:in `evaluate_selection_with_args'
	from /Users/adrien/.rbenv/versions/3.0.2/lib/ruby/gems/3.0.0/gems/graphql-1.12.21/lib/graphql/execution/interpreter/runtime.rb:425:in `evaluate_selection'
	from /Users/adrien/.rbenv/versions/3.0.2/lib/ruby/gems/3.0.0/gems/graphql-1.12.21/lib/graphql/execution/interpreter/runtime.rb:358:in `block (2 levels) in evaluate_selections'
	from /Users/adrien/.rbenv/versions/3.0.2/lib/ruby/gems/3.0.0/gems/graphql-1.12.21/lib/graphql/dataloader/null_dataloader.rb:17:in `append_job'
	from /Users/adrien/.rbenv/versions/3.0.2/lib/ruby/gems/3.0.0/gems/graphql-1.12.21/lib/graphql/execution/interpreter/runtime.rb:357:in `block in evaluate_selections'
	from /Users/adrien/.rbenv/versions/3.0.2/lib/ruby/gems/3.0.0/gems/graphql-1.12.21/lib/graphql/execution/interpreter/runtime.rb:356:in `each'
	from /Users/adrien/.rbenv/versions/3.0.2/lib/ruby/gems/3.0.0/gems/graphql-1.12.21/lib/graphql/execution/interpreter/runtime.rb:356:in `evaluate_selections'
	from /Users/adrien/.rbenv/versions/3.0.2/lib/ruby/gems/3.0.0/gems/graphql-1.12.21/lib/graphql/execution/interpreter/runtime.rb:730:in `block (3 levels) in continue_field'
	from /Users/adrien/.rbenv/versions/3.0.2/lib/ruby/gems/3.0.0/gems/graphql-1.12.21/lib/graphql/execution/interpreter/runtime.rb:788:in `resolve_with_directives'
	from /Users/adrien/.rbenv/versions/3.0.2/lib/ruby/gems/3.0.0/gems/graphql-1.12.21/lib/graphql/execution/interpreter/runtime.rb:729:in `block (2 levels) in continue_field'
	from /Users/adrien/.rbenv/versions/3.0.2/lib/ruby/gems/3.0.0/gems/graphql-1.12.21/lib/graphql/execution/interpreter/runtime.rb:189:in `tap_or_each'
	from /Users/adrien/.rbenv/versions/3.0.2/lib/ruby/gems/3.0.0/gems/graphql-1.12.21/lib/graphql/execution/interpreter/runtime.rb:720:in `block in continue_field'
	from /Users/adrien/.rbenv/versions/3.0.2/lib/ruby/gems/3.0.0/gems/graphql-1.12.21/lib/graphql/execution/interpreter/runtime.rb:887:in `after_lazy'
	from /Users/adrien/.rbenv/versions/3.0.2/lib/ruby/gems/3.0.0/gems/graphql-1.12.21/lib/graphql/execution/interpreter/runtime.rb:706:in `continue_field'
	from /Users/adrien/.rbenv/versions/3.0.2/lib/ruby/gems/3.0.0/gems/graphql-1.12.21/lib/graphql/execution/interpreter/runtime.rb:517:in `block (3 levels) in evaluate_selection_with_args'
	from /Users/adrien/.rbenv/versions/3.0.2/lib/ruby/gems/3.0.0/gems/graphql-1.12.21/lib/graphql/execution/interpreter/runtime.rb:887:in `after_lazy'
	from /Users/adrien/.rbenv/versions/3.0.2/lib/ruby/gems/3.0.0/gems/graphql-1.12.21/lib/graphql/execution/interpreter/runtime.rb:514:in `block (2 levels) in evaluate_selection_with_args'
	from /Users/adrien/.rbenv/versions/3.0.2/lib/ruby/gems/3.0.0/gems/graphql-1.12.21/lib/graphql/execution/interpreter/runtime.rb:788:in `resolve_with_directives'
	from /Users/adrien/.rbenv/versions/3.0.2/lib/ruby/gems/3.0.0/gems/graphql-1.12.21/lib/graphql/execution/interpreter/runtime.rb:503:in `block in evaluate_selection_with_args'
	from /Users/adrien/.rbenv/versions/3.0.2/lib/ruby/gems/3.0.0/gems/graphql-1.12.21/lib/graphql/execution/interpreter/runtime.rb:887:in `after_lazy'
	from /Users/adrien/.rbenv/versions/3.0.2/lib/ruby/gems/3.0.0/gems/graphql-1.12.21/lib/graphql/execution/interpreter/runtime.rb:437:in `evaluate_selection_with_args'
	from /Users/adrien/.rbenv/versions/3.0.2/lib/ruby/gems/3.0.0/gems/graphql-1.12.21/lib/graphql/execution/interpreter/runtime.rb:429:in `block in evaluate_selection'
	from /Users/adrien/.rbenv/versions/3.0.2/lib/ruby/gems/3.0.0/gems/graphql-1.12.21/lib/graphql/schema/member/has_arguments.rb:162:in `block (3 levels) in coerce_arguments'
	from /Users/adrien/.rbenv/versions/3.0.2/lib/ruby/gems/3.0.0/gems/graphql-1.12.21/lib/graphql/dataloader/null_dataloader.rb:17:in `append_job'
	from /Users/adrien/.rbenv/versions/3.0.2/lib/ruby/gems/3.0.0/gems/graphql-1.12.21/lib/graphql/schema/member/has_arguments.rb:143:in `block (2 levels) in coerce_arguments'
	from /Users/adrien/.rbenv/versions/3.0.2/lib/ruby/gems/3.0.0/gems/graphql-1.12.21/lib/graphql/schema/member/has_arguments.rb:142:in `each'
	from /Users/adrien/.rbenv/versions/3.0.2/lib/ruby/gems/3.0.0/gems/graphql-1.12.21/lib/graphql/schema/member/has_arguments.rb:142:in `block in coerce_arguments'
	from /Users/adrien/.rbenv/versions/3.0.2/lib/ruby/gems/3.0.0/gems/graphql-1.12.21/lib/graphql/schema/member/has_arguments.rb:171:in `coerce_arguments'
	from /Users/adrien/.rbenv/versions/3.0.2/lib/ruby/gems/3.0.0/gems/graphql-1.12.21/lib/graphql/execution/interpreter/arguments_cache.rb:50:in `dataload_for'
	from /Users/adrien/.rbenv/versions/3.0.2/lib/ruby/gems/3.0.0/gems/graphql-1.12.21/lib/graphql/execution/interpreter/runtime.rb:428:in `evaluate_selection'
	from /Users/adrien/.rbenv/versions/3.0.2/lib/ruby/gems/3.0.0/gems/graphql-1.12.21/lib/graphql/execution/interpreter/runtime.rb:358:in `block (2 levels) in evaluate_selections'
	from /Users/adrien/.rbenv/versions/3.0.2/lib/ruby/gems/3.0.0/gems/graphql-1.12.21/lib/graphql/dataloader/null_dataloader.rb:17:in `append_job'
	from /Users/adrien/.rbenv/versions/3.0.2/lib/ruby/gems/3.0.0/gems/graphql-1.12.21/lib/graphql/execution/interpreter/runtime.rb:357:in `block in evaluate_selections'
	from /Users/adrien/.rbenv/versions/3.0.2/lib/ruby/gems/3.0.0/gems/graphql-1.12.21/lib/graphql/execution/interpreter/runtime.rb:356:in `each'
	from /Users/adrien/.rbenv/versions/3.0.2/lib/ruby/gems/3.0.0/gems/graphql-1.12.21/lib/graphql/execution/interpreter/runtime.rb:356:in `evaluate_selections'
	from /Users/adrien/.rbenv/versions/3.0.2/lib/ruby/gems/3.0.0/gems/graphql-1.12.21/lib/graphql/execution/interpreter/runtime.rb:230:in `block (4 levels) in run_eager'
	from /Users/adrien/.rbenv/versions/3.0.2/lib/ruby/gems/3.0.0/gems/graphql-1.12.21/lib/graphql/execution/interpreter/runtime.rb:788:in `resolve_with_directives'
	from /Users/adrien/.rbenv/versions/3.0.2/lib/ruby/gems/3.0.0/gems/graphql-1.12.21/lib/graphql/execution/interpreter/runtime.rb:229:in `block (3 levels) in run_eager'
	from /Users/adrien/.rbenv/versions/3.0.2/lib/ruby/gems/3.0.0/gems/graphql-1.12.21/lib/graphql/dataloader/null_dataloader.rb:17:in `append_job'
	from /Users/adrien/.rbenv/versions/3.0.2/lib/ruby/gems/3.0.0/gems/graphql-1.12.21/lib/graphql/execution/interpreter/runtime.rb:227:in `block (2 levels) in run_eager'
	from /Users/adrien/.rbenv/versions/3.0.2/lib/ruby/gems/3.0.0/gems/graphql-1.12.21/lib/graphql/execution/interpreter/runtime.rb:189:in `tap_or_each'
	from /Users/adrien/.rbenv/versions/3.0.2/lib/ruby/gems/3.0.0/gems/graphql-1.12.21/lib/graphql/execution/interpreter/runtime.rb:218:in `block in run_eager'
	from /Users/adrien/.rbenv/versions/3.0.2/lib/ruby/gems/3.0.0/gems/graphql-1.12.21/lib/graphql/execution/interpreter/runtime.rb:788:in `resolve_with_directives'
	from /Users/adrien/.rbenv/versions/3.0.2/lib/ruby/gems/3.0.0/gems/graphql-1.12.21/lib/graphql/execution/interpreter/runtime.rb:209:in `run_eager'
	from /Users/adrien/.rbenv/versions/3.0.2/lib/ruby/gems/3.0.0/gems/graphql-1.12.21/lib/graphql/execution/interpreter.rb:73:in `block in evaluate'
	from /Users/adrien/.rbenv/versions/3.0.2/lib/ruby/gems/3.0.0/gems/graphql-1.12.21/lib/graphql/tracing.rb:66:in `trace'
	from /Users/adrien/.rbenv/versions/3.0.2/lib/ruby/gems/3.0.0/gems/graphql-1.12.21/lib/graphql/execution/interpreter.rb:72:in `evaluate'
	from /Users/adrien/.rbenv/versions/3.0.2/lib/ruby/gems/3.0.0/gems/graphql-1.12.21/lib/graphql/execution/interpreter.rb:48:in `begin_query'
	from /Users/adrien/.rbenv/versions/3.0.2/lib/ruby/gems/3.0.0/gems/graphql-1.12.21/lib/graphql/execution/multiplex.rb:85:in `begin_query'
	from /Users/adrien/.rbenv/versions/3.0.2/lib/ruby/gems/3.0.0/gems/graphql-1.12.21/lib/graphql/execution/multiplex.rb:104:in `block (2 levels) in run_as_multiplex'
	from /Users/adrien/.rbenv/versions/3.0.2/lib/ruby/gems/3.0.0/gems/graphql-1.12.21/lib/graphql/dataloader/null_dataloader.rb:17:in `append_job'
	from /Users/adrien/.rbenv/versions/3.0.2/lib/ruby/gems/3.0.0/gems/graphql-1.12.21/lib/graphql/execution/multiplex.rb:104:in `block in run_as_multiplex'
	from /Users/adrien/.rbenv/versions/3.0.2/lib/ruby/gems/3.0.0/gems/graphql-1.12.21/lib/graphql/execution/multiplex.rb:103:in `each'
	from /Users/adrien/.rbenv/versions/3.0.2/lib/ruby/gems/3.0.0/gems/graphql-1.12.21/lib/graphql/execution/multiplex.rb:103:in `each_with_index'
	from /Users/adrien/.rbenv/versions/3.0.2/lib/ruby/gems/3.0.0/gems/graphql-1.12.21/lib/graphql/execution/multiplex.rb:103:in `run_as_multiplex'
	from /Users/adrien/.rbenv/versions/3.0.2/lib/ruby/gems/3.0.0/gems/graphql-1.12.21/lib/graphql/execution/multiplex.rb:63:in `block (2 levels) in run_queries'
	from /Users/adrien/.rbenv/versions/3.0.2/lib/ruby/gems/3.0.0/gems/graphql-1.12.21/lib/graphql/execution/multiplex.rb:208:in `block in instrument_and_analyze'
	from /Users/adrien/.rbenv/versions/3.0.2/lib/ruby/gems/3.0.0/gems/graphql-1.12.21/lib/graphql/execution/instrumentation.rb:29:in `block (2 levels) in apply_instrumenters'
	from /Users/adrien/.rbenv/versions/3.0.2/lib/ruby/gems/3.0.0/gems/graphql-1.12.21/lib/graphql/execution/instrumentation.rb:46:in `block (2 levels) in each_query_call_hooks'
	from /Users/adrien/.rbenv/versions/3.0.2/lib/ruby/gems/3.0.0/gems/graphql-1.12.21/lib/graphql/execution/instrumentation.rb:41:in `each_query_call_hooks'
	from /Users/adrien/.rbenv/versions/3.0.2/lib/ruby/gems/3.0.0/gems/graphql-1.12.21/lib/graphql/execution/instrumentation.rb:45:in `block in each_query_call_hooks'
	from /Users/adrien/.rbenv/versions/3.0.2/lib/ruby/gems/3.0.0/gems/graphql-1.12.21/lib/graphql/execution/instrumentation.rb:72:in `call_hooks'
	from /Users/adrien/.rbenv/versions/3.0.2/lib/ruby/gems/3.0.0/gems/graphql-1.12.21/lib/graphql/execution/instrumentation.rb:44:in `each_query_call_hooks'
	from /Users/adrien/.rbenv/versions/3.0.2/lib/ruby/gems/3.0.0/gems/graphql-1.12.21/lib/graphql/execution/instrumentation.rb:27:in `block in apply_instrumenters'
	from /Users/adrien/.rbenv/versions/3.0.2/lib/ruby/gems/3.0.0/gems/graphql-1.12.21/lib/graphql/execution/instrumentation.rb:72:in `call_hooks'
	from /Users/adrien/.rbenv/versions/3.0.2/lib/ruby/gems/3.0.0/gems/graphql-1.12.21/lib/graphql/execution/instrumentation.rb:26:in `apply_instrumenters'
	from /Users/adrien/.rbenv/versions/3.0.2/lib/ruby/gems/3.0.0/gems/graphql-1.12.21/lib/graphql/execution/multiplex.rb:187:in `instrument_and_analyze'
	from /Users/adrien/.rbenv/versions/3.0.2/lib/ruby/gems/3.0.0/gems/graphql-1.12.21/lib/graphql/execution/multiplex.rb:62:in `block in run_queries'
	from /Users/adrien/.rbenv/versions/3.0.2/lib/ruby/gems/3.0.0/gems/graphql-1.12.21/lib/graphql/tracing.rb:66:in `trace'
	from /Users/adrien/.rbenv/versions/3.0.2/lib/ruby/gems/3.0.0/gems/graphql-1.12.21/lib/graphql/execution/multiplex.rb:60:in `run_queries'
	from /Users/adrien/.rbenv/versions/3.0.2/lib/ruby/gems/3.0.0/gems/graphql-1.12.21/lib/graphql/execution/multiplex.rb:50:in `run_all'
	from /Users/adrien/.rbenv/versions/3.0.2/lib/ruby/gems/3.0.0/gems/graphql-1.12.21/lib/graphql/schema.rb:1701:in `multiplex'
	from /Users/adrien/.rbenv/versions/3.0.2/lib/ruby/gems/3.0.0/gems/graphql-1.12.21/lib/graphql/schema.rb:1672:in `execute'
	from /Users/adrien/workspace/sorare/backend/spec/graphql/invalid_cursor.rb:51:in `<main>'
```

</details>

**Comment by rmosolgo** [comment]
Thanks for the detailed report, and sorry for the trouble! I've just released GraphQL-Pro 1.20.3 which handles this cases with a user-facing error: `Invalid cursor for `after: "WyIyNmEzODU1YS0"`. Fetch a new cursor and try again.`

Out of curiosity, do you know where that invalid cursor _came_ from? Obviously I hadn't prepared for this case, so I'm interested to learn how it came up.

**Comment by such** [comment]
Hey,

Thank you for the quick turnaround!

The invalid cursor came from a user of our API. No idea where he got it. My
guess is that he uses a buggy implementation of graphql and truncated the
cursor...

Best,

Adrien
